edx_proctoring/backends/rest.py

```diff
@@ -266,8 +266,11 @@ def on_exam_saved(self, exam):
             response = self.session.post(url, json=exam)
             data = response.json()
         except Exception as exc:  # pylint: disable=broad-except
```

Comment:
could we branching below by tracing down the specific exception types that would be thrown here? I'm not sure we ever had a compelling reason to relax this lint rule, and it seems like it's related to how this issue came to be

Reply:
It's a broad except because it's not clear which exceptions could be thrown. In this case, it was a `TypeError` from json serialization. An exception already would've been logged in splunk.

Reply:
Okay, could we add the branching logic as a more-specific except listed above this one and eliminate the `if`/`else` that way?